utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/hostapd_stations.lua

```diff
@@ -1,90 +1,117 @@
 local ubus = require "ubus"
 local bit32 = require "bit32"
 
-local function get_wifi_interfaces()
+local function get_wifi_interface_labels()
   local u = ubus.connect()
   local status = u:call("network.wireless", "status", {})
   local interfaces = {}
 
   for _, dev_table in pairs(status) do
     for _, intf in ipairs(dev_table['interfaces']) do
-      table.insert(interfaces, intf['ifname'])
+      local cfg = intf['config']
+
+      -- Migrate this to ubus interface once it exposes all interesting labels
```

Comment:
https://patchwork.ozlabs.org/project/openwrt/patch/20210625220108.151779-1-hexa@darmstadt.ccc.de/

Wrote some code to that end.

Reply:
This just got merged https://git.openwrt.org/?p=openwrt/openwrt.git;a=commitdiff;h=398df6275674b637bc950e4f946f84c2bc5e09ff